agent/pool/pool.go

```diff
@@ -607,14 +607,14 @@ func (p *ConnPool) rpcInsecure(dc string, addr net.Addr, method string, args int
 	var codec rpc.ClientCodec
 	conn, _, err := p.dial(dc, addr, 0, RPCTLSInsecure)
 	if err != nil {
```

Comment:
I thought adding `:`'s here would be nice because `rpcinsecure` is meant to indicate the area the code is in. It's not an "rpcinsecure error" which makes me think rpcinsecure is having an issue, it's an error while making an rpcinsecure call.